sys/include/net/gnrc/sixlowpan/frag.h

```diff
@@ -94,6 +97,13 @@ typedef struct {
     uint16_t tag;           /**< Tag used for the fragment */
     uint16_t offset;        /**< Offset of the Nth fragment from the beginning of the
                              *   payload datagram */
+#ifdef MODULE_GNRC_SIXLOWPAN_FRAG_HINT
+    /**
+     * @brief   Hint for the size (smaller than link-layer PDU) of the
+     *          *first fragment*
```

Comment:
Why **first** fragment? The code seems to indicate something else (for all fragments?). Also, other parts of the documentation also refer to more fragments than just the first.

Reply:
I think I meant the first fragment that is handled by this... But I just remove it, as it is indeed confusing.